episodes/4-advanced-layer-types.Rmd

```diff
@@ -471,35 +467,35 @@ dense_model.summary()
 ```output
 Model: "dense_model"
 _________________________________________________________________
- Layer (type)                Output Shape              Param #
+ Layer (type)                Output Shape              Param #   
 =================================================================
- input_9 (InputLayer)        [(None, 32, 32, 3)]       0
-
- flatten_7 (Flatten)         (None, 3072)              0
-
- dense_21 (Dense)            (None, 50)                153650
-
- dense_22 (Dense)            (None, 50)                2550
-
- dense_23 (Dense)            (None, 10)                510
-
+ input_11 (InputLayer)       [(None, 128, 128, 3)]     0         
+                                                                 
+ flatten_10 (Flatten)        (None, 49152)             0         
+                                                                 
+ dense_25 (Dense)            (None, 50)                2457650   
+                                                                 
+ dense_26 (Dense)            (None, 50)                2550      
+                                                                 
+ dense_27 (Dense)            (None, 10)                510       
+                                                                 
 =================================================================
-Total params: 156710 (612.15 KB)
-Trainable params: 156710 (612.15 KB)
+Total params: 2460710 (9.39 MB)
+Trainable params: 2460710 (9.39 MB)
 Non-trainable params: 0 (0.00 Byte)
 _________________________________________________________________
 ```
-As you can see this model has 1.5x more parameters than our simple CNN, let's train and evaluate it!
+As you can see this model has more parameters than our simple CNN, let's train and evaluate it!
 
 ```python
 compile_model(dense_model)
-history = dense_model.fit(train_images, train_labels, epochs=30,
+history = dense_model.fit(train_images, train_labels, epochs=20,
```

Comment:
A couple of observations when I got to this point:

1. This model trained _very_ quickly
2. The training history plot looks weird...

![04_dense_model_training_history](https://github.com/carpentries-incubator/deep-learning-intro/assets/9694524/507b16da-4734-4b1c-9e49-758a7c251a96)